dockerfiles/run/guice/cassandra-ldap/Dockerfile

```diff
@@ -27,5 +27,10 @@ ADD destination/conf /root/conf
 
 VOLUME /logs
 VOLUME /root/conf
+VOLUME /root/included-jars
 
-ENTRYPOINT java -Dlogback.configurationFile=/root/conf/logback.xml -Dworking.directory=/root/ -jar james-server.jar 
+ENTRYPOINT java \
+  -classpath '/root/james-server.jar:/root/james-server-cassandra-ldap-guice.lib/*:/root/included-jars/*' \
```

Comment:
what I mean is that you are repeating what is inside the manifest, thus, the manifest becomes useless.
I think it's a regression. Maybe it's the price to pay (I doubt), but then we would have to remove the code to generate this manifest.